client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -400,7 +401,7 @@ export default {
         params.assigneeId = this.actions.assignSegment.selected.id
       }
 
-      dpRpc('segment.bulk.edit', params)
+      dpRpc('segment.bulk.edit', params, 'rpc_generic_post')
```

Comment:
why is the 'rpc_generic_post' not part of the params?
here for the general Question

Reply:
Revert here: https://github.com/demos-europe/demosplan-core/pull/718/commits/13bee5e02e09e441de4afd53ee38456e77bfe7c3